datatypes/src/raster/macros_raster.rs

```diff
@@ -0,0 +1,65 @@
+/// Maps a `TypedGridArray2D` to another `TypedGridArray2D` by calling a function on its variant.
+/// Call via `map_generic_raster2d!(input, raster => function)`.
+#[macro_export]
+macro_rules! map_generic_raster_2d {
```

Comment:
why are the macros called ...raster... if they operate of TypedGridArrays?

Reply:
will do when renaming GridArray

Reply:
done